scripts/linters/excluded_any_type_files.json

```diff
@@ -5,15 +5,17 @@
     "core/templates/domain/collection/read-only-collection-backend-api.service.ts",
     "core/templates/domain/collection/search-explorations-backend-api.service.ts",
     "core/templates/domain/exploration/ExplorationMetadataObjectFactory.ts",
+    "core/templates/domain/exploration/InteractionObjectFactory.ts",
```

Comment:
I don't believe it's OK to add new files to this -- @nishantwrp, could you please advise?

Reply:
Yup, I couldn't see the `any` in InteractionObjectFactory. Why did you add this?

Reply:
@shavavo plesae note this comment.

Reply:
Sorry for missing this. InteractionObjectFactory no longer uses any, and excluded_any_type_files.json file has since been deleted from a merge from develop. Done.